FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -469,11 +469,16 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <locus from="122rb" to="122vb"/>
                         <locus from="129ra" to="131ra"/>
                         <title type="incomplete" ref="LIT2382Taamme">Miracles of Jesus, 3 stories.</title>
+                        <note>Names of new owners <persName ref="PRS13971Abagya">ዓበግያ</persName> and her father 
```

Comment:
add `role="owner"` inside `persName`

Reply:
Done for Abagya. I will do this also for Kenfa Mikael, Gabra Zara Syon and Fesseha Giyorgis.